.mergify.yml

```diff
@@ -1,3 +1,13 @@
+shared:
+  - commit_message_template: &commit_message_template |
+      {{title}}
+      
+      {{body | get_section("## What's changed and what's your intention?")}}
```

Comment:
Considering many ppl don't keep the section very clean, it's still a little noisy. But maybe acceptable. 😇

Reply:
It's another reason to urge us to write a cleaner "What's changed" description. Let me announce it again in public channels 🤣

Reply:
What'll happen if we paste an image here?

Reply:
Good question @TennyZhuang. We should definitely try this before merging this PR

Reply:
Perhaps you can only try it after this PR being merged 😇